components/style/properties/longhand/font.mako.rs

```diff
@@ -1835,11 +1835,12 @@ ${helpers.single_keyword_system("font-variant-position",
         impl ToCss for FeatureTagValue {
             fn to_css<W>(&self, dest: &mut W) -> fmt::Result where W: fmt::Write {
                 use std::str;
-                use byteorder::{WriteBytesExt, BigEndian};
+                use byteorder::{BigEndian, ByteOrder};
                 use cssparser::serialize_string;
+                use smallvec::SmallVec;
 
-                let mut raw: Vec<u8> = vec!();
-                raw.write_u32::<BigEndian>(self.tag).unwrap();
+                let mut raw = [0u8; 4];
+                BigEndian::write_u32(&mut raw, (self.tag));
```

Comment:
I thought unwrap was being taken care of by the fn serialize_string() below it. The parenthesis are just a bad habit I've picked up on :). Trying to unwrap() self.tag now is giving me the an error:  no method named 'unwrap' found for type 'u32' in current scope. Not sure why.